pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -335,13 +335,12 @@ func (gc *GarbageCollector) attemptToDeleteWorker(ctx context.Context) bool {
 	}
 
 	err := gc.attemptToDeleteItem(ctx, n)
-	if err == enqueuedVirtualDeleteEventErr {
+	switch {
+	case err == enqueuedVirtualDeleteEventErr:
```

Comment:
Why don't we forget in this case?  The sync loop here didn't have an error

Reply:
Got it, updated. PTAL thanks.